EnglishFilter/sections/general_extensions.txt

```diff
@@ -234,6 +234,9 @@ flashx.net#%#//scriptlet("abort-current-inline-script", "decodeURIComponent", "S
 cambay.tv#%#//scriptlet("set-constant", "flashvars.popunder_url", "")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69639
 fappee.com#%#//scriptlet('abort-current-inline-script', 'doMyStuff')
+! https://github.com/AdguardTeam/AdguardFilters/issues/70037
+iceporn.com#%#("abort-on-property-write", "mobileAdvPop")
+iceporn.com#%#document.cookie = "adv_show=10;";
```

Comment:
It seems that similar rule is already in `AdGuard Mobile Ads filter`, do we need it also for not mobile website?
If so, maybe it would be better to use `set-cookie` scriptlet, so rule will be available in Firefox and Opera extension too.

By the way, I see popups on not mobile website, something like that should block them:
`iceporn.com#%#//scriptlet("prevent-window-open", "1", "about:blank")`

Reply:
Changed `document.cookie` to `set-cookie` scriptlet.

Strangely, i experience popups without "mobileAdvPop" rule on desktop version too, and "prevent-window-open doesnt work for me".